package.json

```diff
@@ -14,7 +14,7 @@
     "validate": "eslint --ext .js ./src ./env ./server ./test",
     "lintfix": "eslint --ext .js ./src ./env ./server ./test --fix",
 
-    "build": "webpack --progress --profile --config ./env/webpack.prod.config.js",
+    "build": "cross-env ENV_CONFIG=production NODE_ENV=production webpack --progress --profile -p --config ./env/webpack.prod.config.js",
```

Comment:
I've check NODE_ENV=production - does not make effect in this project, so removed that. 
New task "build: production" - were created as suggested.